From 2a5344234bf01bf3d43d4ba81f997cc181e0237a Mon Sep 17 00:00:00 2001
From: Jason Kridner <jdk@ti.com>
Date: Thu, 15 May 2008 06:11:29 -0500
Subject: [PATCH] Just a dummy test to understand patch submittal.


Signed-off-by: Jason Kridner <jdk@ti.com>
---
 README |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/README b/README
index cc1fc9f..0f8776d 100644
--- a/README
+++ b/README
@@ -8,3 +8,4 @@ I was unable to use WebDAV on Amazon EC2 to allow pushing the git tree for this
 I have instead enabled ssh pushes at ssh://beagleboard.org/var/git/beagleboard.org.git/.  The
 source can be cloned at http://www.beagleboad.org/beagleboard.org.git/.
 
+A change.
-- 
1.5.4.3

